gradle.properties

```diff
@@ -25,8 +25,8 @@ android.useAndroidX=true
 android.enableJetifier=true
 
 GROUP=com.amazonaws
-VERSION_NAME=2.22.2
-VERSION_CODE=022202
+VERSION_NAME=2.22.3
+VERSION_CODE=022203
```

Comment:
```suggestion
```

Reply:
@richardmcclellan, do we not use the version code anymore?

Reply:
@raphkim we [removed it from amplify-android](https://github.com/aws-amplify/amplify-android/pull/1223), and the change was released yesterday.    My understanding is that it is not needed at all for an Android library - it's only needed for an Android app.

Reply:
If we want to remove `VERSION_CODE`, I think it's better to put another PR.

Reply:
@changxu0306 okay! https://github.com/aws-amplify/aws-sdk-android/pull/2388

Reply:
I just checked the [AppSync repo's `gradle.properties`](https://github.com/awslabs/aws-mobile-appsync-sdk-android/blob/main/gradle.properties), and it doesn't even have `VERSION_CODE`!    So, that further confirms removing it is fine.